addons/pos_self_order/tests/test_self_order_sequence.py

```diff
@@ -0,0 +1,43 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import odoo.tests
+
+
+@odoo.tests.tagged("post_install", "-at_install")
+class TestSelfOrderSequence(odoo.tests.HttpCase):
```

Comment:
Why don't you inherit from SelfOrderCommonTest, the pos_user, pos_admin and pos_config would already be created. Also when you do not use Common test class you can have some errors linked to the usage of some demo data instead of test data like here (you use Coca-Cola without creating it first here).

Reply:
SelfOrderCommonTest modifies the browser size which makes OrderNumberConflictTour fails because it has to be run in normal browser size. I am not really knowledgeable in this so I don't know if I can modify the browser size just before a specific tour gets triggered.